### PR TITLE
Updating Kong EE and Kong CE to new name scheme

### PR DESCRIPTION
**Pull request by john-paul-keenan**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

SUMMARY_GOES_HERE

### Full changelog

* Changed every instance of Kong Community Edition, Kong CE and Kong (CE) to "Kong"
* Changed every instance of Kong EE or Kong Enterprise Edition to "Kong Enterprise"
* Made sure the content still made sense with the updates

### Issues resolved

The new official names of the products are Kong and Kong Enterprise. I went through and corrected all appearances of the names to reflect this change.

### Checklist:

- [ ] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [ ] Documentation 
- [ ] Spellchecked my updates
- [ ] Ready to be merged

### CODE REVIEW

File: app/_layouts/docs.html [hunk 1]
```diff
@@ -109,7 +109,7 @@ <h5>{{nav.title}}</h5>
         <div class="content">
           {% if page.edition == 'community' %}
             <div class="alert alert-ee">
-              Maybe you were looking for the <a href="/enterprise"><strong>Enterprise Edition (EE) Documentation</strong></a> instead?
+              Maybe you were looking for the <a href="/enterprise"><strong>Enterprise Documentation</strong></a> instead?
```
Comment (coopr) [inline app/_layouts/docs.html]: "Maybe you were looking for the Kong Enterprise documentation instead?" sounds better

File: app/_layouts/extension.html [hunk 2]
```diff
@@ -217,7 +217,7 @@
 
       {% if page.enterprise %}
         <div class="alert alert-warning">
-          <strong>Kong Enterprise Edition (EE) only</strong>: This plugin is only available with a <a href="https://konghq.com/kong-enterprise-edition">Kong EE subscription</a>.
+          <strong>Kong Enterprise only</strong>: This plugin is only available with a <a href="https://konghq.com/kong-enterprise-edition">Kong EE subscription</a>.
```
Comment (coopr) [inline app/_layouts/extension.html]: EE is mentioned here

File: app/enterprise/0.31-x/edition-versioning.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
 ---
-title: Kong Enterprise Edition Versioning
+title: Kong Enterprise Versioning
 ---
 
-Kong Enterprise Edition is distributed as a bundle of elements, including:
+Kong Enterprise is distributed as a bundle of elements, including:
 
-- **Kong Core**, built from Kong Community Edition
+- **Kong Core**, built from Kong
```
Comment (coopr) [inline app/enterprise/0.31-x/edition-versioning.md]: Ugh, this isn't terribly clear - I'm not sure what to suggest to clarify it.

File: app/enterprise/0.31-x/getting-started/adding-your-api.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ title: Adding your API
 
 # Adding your API
 
-In this section, you'll be adding your API to Kong Enterprise Edition (EE).
+In this section, you'll be adding your API to Kong Enterprise.
 This is the first
 step to having Kong EE manage your API. For purposes of this Getting Started
```
Comment (coopr) [inline app/enterprise/0.31-x/getting-started/adding-your-api.md]: EE is mentioned

File: app/enterprise/0.31-x/getting-started/licensing.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 ---
 
-# Kong Enterprise Edition Licensing
+# Kong Enterprise Licensing
 
 ## Overview
-Kong Enterprise Edition enforces the presence and validity of a license file. 
+Kong Enterprise enforces the presence and validity of a license file. 
 
 License files must be deployed to each node running Kong EE. License file checking is done independently by each node as the Kong process starts; no network connectivity is necessary to execute the license validation process.
```
Comment (coopr) [inline app/enterprise/0.31-x/getting-started/licensing.md]: EE is mentioned

File: app/enterprise/0.31-x/index.md [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
+# Welcome to Kong Enterprise
 
 {% if page.edition == 'enterprise' %}
 <div class="alert alert-ee">
   <div class="alert-body">
     <div class="left">
       <img src="/assets/images/icons/icn-buildings.svg" />
     </div>
-    <p>Kong Enterprise Edition (EE) adds features, functionality, and performance to Kong Community Edition (CE). This documentation doesn’t cover the general practices of using Kong that are common to both Kong CE and EE - learn the <a href="">basics in Kong CE documentation</a>.</p>
+    <p>Kong Enterprise adds features, functionality, and performance to Kong. This documentation doesn’t cover the general practices of using Kong that are common to both Kong and EE - learn the <a href="">basics in Kong documentation</a>.</p>
```
Comment (coopr) [inline app/enterprise/0.31-x/index.md]: EE is mentioned.

I'll stop my review here and await a revised PR, thanks!

Comment (coopr) [reply app/enterprise/0.31-x/index.md]: I _think_ you could resolve most of the remaining EE mentions by searching for "EE" and replacing it with "Kong Enterprise" - but you'd want to review those as you go, just to be sure.

**Comment by john-paul-keenan** [comment]
Went through and fixed a bunch more. Not sure if I got all of them, will need to check later this week.

**Comment by coopr** [comment]
@john-paul-keenan please review and resolve conversations herein, and also please ping me again when you do further review/improvements - thanks!

**Comment by coopr** [comment]
@john-paul-keenan do you plan to resolve the merge conflicts? Or is this now too out of date, and you plan to close it? TIA!

**Pull request closed without merging**